proto/ory/keto/relation_tuples/v1alpha2/check_service.proto

```diff
@@ -35,9 +74,26 @@ message CheckRequest {
   // The relation between the Object and the Subject.
   string relation = 3 [deprecated = true];
   // The related subject in this check.
-  Subject subject = 4 [deprecated = true];
+  Subject subject = 4 [
+    deprecated = true,
+    (google.api.field_visibility).restriction = "NO_SWAGGER"
```

Comment:
This results in the fields not included in the openapi spec?

Reply:
Yes, correct.